src/pages/create-or-edit-lesson/CreateOrEditLesson.tsx

```diff
@@ -250,25 +248,22 @@ const CreateOrEditLesson = () => {
           onCategoryChange={onCategoryChange}
         />
         <Divider sx={styles.divider} />
-        <AppButton
+        <Button
+          endIcon={<AddIcon sx={styles.addIcon} />}
           onClick={handleOpenAddAttachmentsModal}
           sx={styles.addAttachments}
         >
-          {t('lesson.labels.attachments')} <AddIcon sx={styles.addIcon} />
-        </AppButton>
+          {t('lesson.labels.attachments')}
+        </Button>
         <FileEditor onEdit={handleEdit} value={data.content} />
         {attachmentsList}
         <Box sx={styles.buttons}>
-          <AppButton
-            onClick={navigateToLessonTab}
-            size={SizeEnum.XXL}
-            variant={ButtonVariantEnum.Tonal}
-          >
+          <Button onClick={navigateToLessonTab} size='md' variant='tonal'>
             {t('common.cancel')}
-          </AppButton>
-          <AppButton size={SizeEnum.XXL} type={ButtonTypeEnum.Submit}>
+          </Button>
+          <Button size='md' type={ButtonTypeEnum.Submit}>
```

Comment:
somewhere before you wrote `type=‘submit’`, you can change it using the enum like here

Reply:
changed all to enum